fastlane/docs/Plugins.md

```diff
@@ -56,7 +56,8 @@ This will:
 
 - Add the plugin to `fastlane/Pluginfile`
 - Make sure your `fastlane/Pluginfile` is properly referenced from your `./Gemfile`
-- Run `bundle install` to make sure all required dependencies are installed on your local machine (this step might ask for your admin password to install Ruby gems)
+- Run `fastlane install_dependencies` to make sure all required dependencies are installed on your local machine (this step might ask for your admin password to install Ruby gems)
```

Comment:
The method name is `install_dependencies`, but the fastlane command name is `install_plugins`, right?